docs/platform/linode-managed-classic-manager/index.md

```diff
@@ -10,7 +10,6 @@ modified_by:
 published: 2013-02-12
 cloud_manager_link: platform/linode-managed/
 title: Linode Managed
-cloud_manager_link: platform/linode-managed/
```

Comment:
This actually already had the cloud manager link, so I'm just removing the second instance of this